grpc/proto/v1/properties.proto

```diff
@@ -26,11 +26,49 @@ message Value {
     string blob_value = 10;
     PhoneNumber phone_value = 11;
     google.protobuf.NullValue null_value = 12;
+    string text_value = 13;
   }
 }
 
 message ListValue {
-  repeated Value values = 1;
+  repeated Value values = 1 [deprecated = true];
+  oneof kind {
+    NumberValues number_values = 2;
+    TextValues text_values = 3;
+    BoolValues bool_values = 4;
+    ObjectValues object_values = 5;
+    DateValues date_values = 6;
+    UuidValues uuid_values = 7;
+    IntValues int_values = 8;
+  }
+}
+
+message NumberValues {
+  repeated double values = 1;
```

Comment:
please use bytes for floats - might make sense to reuse the definitions in grpc/proto/v1/base.proto